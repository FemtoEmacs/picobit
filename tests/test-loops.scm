;; tests with loops

(display (map (lambda (x) (+ x 1)) '(1 2 3 4 5)))
(display (for-each (lambda (x) (set! x (+ x 1))) '(1 2 3 4 5)))
