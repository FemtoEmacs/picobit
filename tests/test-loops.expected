:0080000080
:10800000FBD7220280050124811D4000811240001F
:10801000810E4000810C4000810D40008007011955
:10802000804200028027011A8027011B8024011C46
:108030008071011E806E011F8009000280270120CF
:10804000807701218073012280240002806A00026F
:10805000807301238067012580080111804001138E
:1080600080400114803201158027011680660117B7
:108070008069011880780002807D010F8066011000
:108080008070010B8006010A8078010B02B401B5F3
:108090005002B401AF5102B401AAA104B00199B5D4
:1080A0008DEE02B40178A104B58EB582EED8012020
:1080B000DAB8954121DBB6800221CC9A21CE21612C
:1080C000EE2022CFB67200EF20B6800120DA9EA00B
:1080D0002607E620B558EEA02607E600EF20C0B838
:1080E000BF20B582B648012004C8B8AEA03121B582
:1080F000A402B583CDDCEF02A034210EC5C122CD90
:108100002102CDB401077101210EC89220EF210E8A
:10811000406221B662010421C2EF2002B559B75571
:1081200020CCB8B8A02C07E620CEB51FEE20CFB6E5
:10813000800120D296A02D07E600EF20CC9DA02440
:1081400007E620CEB505EE20CFB666A107B000AE9B
:10815000EE20B000CBEEA02D07E600EF20D995A1D0
:1081600008B100AE20F09C2095A106B100AEA1059B
:10817000B100AEA109B100AE0120B59802B40181F1
:1081800071012021B58A05C1B680022022D000EFFE
:108190000120CEEF012002CDEF0221CC9B21CE2188
:1081A000612123CFB573CDEF02EF012005C1EF01AF
:1081B0002007E600EF022004C8212302CDCDB40140
:1081C000C271012204C822242302CDCDCDB401D135
:1081D000710123B8DC21B0011523222602CDCDCDBB
:1081E000B401E4710122B8C621B001152124260290
:1081F000CDCDCDB401F771012022242602CDCDCD05
:10820000CDB4020570002121C4232502CDCDB402D6
:1082100012710122B893219220EF2298019420B18B
:10822000011520EF21B87BB77500B86EB76A21B78A
:048230003B21B72512
:00000001FF
